internal/database/metadata/postgres/entity.go

```diff
@@ -73,11 +74,21 @@ func getEntityByName(ctx context.Context, sqlxCtx metadata.SqlxContext, name str
 
 }
 
-func listEntity(ctx context.Context, sqlxCtx metadata.SqlxContext) (types.EntityList, error) {
-	query := "select * from entity"
+func listEntity(ctx context.Context, sqlxCtx metadata.SqlxContext, entityIDs *[]int) (types.EntityList, error) {
+	query := "SELECT * FROM entity"
+	var args []interface{}
+	var err error
+	if entityIDs != nil {
+		if len(*entityIDs) == 0 {
+			return nil, nil
+		}
```

Comment:
Maybe just checking `len(entityIDs) == 0` will work 🤔

Reply:
it does not work, https://go.dev/play/p/tsALR2HcDmr